app/src/main/java/com/willowtree/vocable/utils/LocalizedResourceUtility.kt

```diff
@@ -0,0 +1,46 @@
+package com.willowtree.vocable.utils
+
+import android.content.Context
+import android.content.res.Resources
+import com.willowtree.vocable.room.Category
+import com.willowtree.vocable.room.Phrase
+import org.koin.core.KoinComponent
+import org.koin.core.get
+import java.util.*
+
+class LocalizedResourceUtility(context: Context) : KoinComponent {
+
+    val resources: Resources = get<Context>().resources
+
+    fun getValueLocalePairFromCategory(category: Category): Pair<String, Locale>? {
```

Comment:
Would it be worth having these methods return non-nullable values? I noticed in a few places we're defaulting to the empty string if they return null. Could we do that in the method?

Reply:
I like that!